package.json

```diff
@@ -8,7 +8,7 @@
     "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-import": "^2.27.5",
     "prettier": "^2.8.8",
-    "turbo": "^1.10.4",
+    "turbo": "^1.10.16",
```

Comment:
I upgraded turbo in the hopes that it would stop breaking my terminal...but it didn't. But I went ahead and left it upgraded.